Appraisals

```diff
@@ -3,5 +3,5 @@ appraise 'activerecord-5' do
 end
 
 appraise 'activerecord-6' do
```

Comment:
Should we test with both 6.0 and 6.1?

Reply:
Good point. I updated the appraisals to reflect it now